Kindle file build fails

First of all: awesome work! I discovered this yesterday and you made my day.

This is an informative, low-priority issue. I was just testing your work with a simple ebook I did time ago, and this came up:

```
$ make buildkindle
Downloading kindlegen...
####################################################################################### 100,0%
Building Kindle file...
/bin/sh: 1: cd: can't cd to tmp/./src/
Makefile:65: recipe for target 'build/ebook.mobi' failed
make: *** [build/ebook.mobi] Error 2
```
(Debian Testing "Buster")

Seems to be something with the path. Kinda weird, because `make buildkepub` do similar things and it works.

--- a/Makefile
+++ b/Makefile
@@ -64,6 +64,7 @@ buildkindle: $(KINDLEFILE)
 $(KINDLEFILE): $(EPUBFILE) $(KINDLEGEN)
 	@echo Building Kindle file...
 	@cp -f "$(EPUBFILE)" "$(KINDLEFILE).epub"
+ifneq ($(PNGFILES),)
 	@for current in $(PNGFILES); do \
 		channels=$$(identify -format '%[channels]' "$$current"); \
 		if [[ "$$channels" == "graya" ]]; then \
@@ -74,6 +75,7 @@ $(KINDLEFILE): $(EPUBFILE) $(KINDLEGEN)
 	done
 	@cd "tmp/$(SOURCE)" && zip -Xr9D "../../$(KINDLEFILE).epub" .
 	@rm -rf "tmp/"
+endif
 	@$(KINDLEGEN) "$(KINDLEFILE).epub" -dont_append_source -c1 || exit 0 # -c1 means standard PalmDOC compression. -c2 takes too long but probably makes it even smaller.
 	@rm -f "$(KINDLEFILE).epub"
 	@mv "$(KINDLEFILE).mobi" "$(KINDLEFILE)"

